src/project/internal/thumbnailcreator.cpp

```diff
@@ -0,0 +1,29 @@
+#include "thumbnailcreator.h"
+
+using namespace au::project;
+
+void ThumbnailCreator::onThumbnailCreated(bool success)
+{
+    m_thumbnailCreated.send(success);
+}
+
+muse::async::Channel<std::string> ThumbnailCreator::captureThumbnailRequested() const
```

Comment:
better to use `muse::io::path_t` to indicate what is file path

Reply:
Done.